### ISSUE TITLE
Details viewer tab to display user content

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#288]
They would like to have the option to upload PDF that would be displayed in the Details Viewer. The upload will take place through the experiment creation tab / edit info and the PDF will be displayed ideally with dark background and white font.

**Comment by afonsobspinto** [comment]
We will need to:
- Add a new tab to the 'Add to experiment' dialog which allows us to update PDF's
- Adapt the model and views to associate uploaded PDF files to experiments
- Process the PDF to follow the style guideline (dark background). If I recall correctly we have a python script for this already
- Adapt the Details Viewer tab to include this new PDFs. Assuming the same accordion style format


<details>
  <summary> I would estimate something around</summary>
24 hours  
</details>

Questions:
1. Opposed to the current PDF's, the ones uploaded won't be associated with a population but rather the experiment correct?

**Comment by neuroBazzi** [comment]
yes, all correct (also the answer to question 1). @zsinnema tohughts?

**Comment by zsinnema** [comment]
@afonsobspinto I think you are correct about linking the pdf to the experiment

**Comment by afonsobspinto** [comment]
- Investigate why there are no PDF's showing up @afonsobspinto

**Comment by D-GopalKrishna** [comment]
@neuroBazzi Is there any sample for this PDF type?

**Comment by neuroBazzi** [comment]
@D-GopalKrishna yes we do have the one from last year, let's keep in mind it's a placeholder and that the user should be able to upload PDFs during experiment creation (or add them at a later stage). cc @afonsobspinto 
[Electrophysiological properties of V2a interneurons in the lumbar spinal cord.pdf](https://github.com/MetaCell/salk-interactive-atlas/files/12929512/Electrophysiological.properties.of.V2a.interneurons.in.the.lumbar.spinal.cord.pdf)

**Comment by afonsobspinto** [comment]
@neuroBazzi can we assume / enforce that the PDFs are 1 page only?
If not, how should the UI handle multiple page PDFs?

**Comment by neuroBazzi** [comment]
@afonsobspinto I would not assume they are one page (ie.e. we should add pagination) and yes, the UI should be able to handle multiple uploads

**Comment by afonsobspinto** [comment]
@syamace-metacell to create a new button ('Add details?) and the modal to associate a population and a category (one of Electrophysiological, Behaviour, I/O Mapping) to the pdf to be added.
@Aiga115 to handle the UI
@D-GopalKrishna to create a new attribute for the population custom pdfs and connect it to the details viewer

**Comment by neuroBazzi** [comment]
@afonsobspinto I am not sure I understand what you mean with original PDF, could you paste an example?

**Comment by D-GopalKrishna** [comment]
@neuroBazzi 

Please correct me if I am wrong @afonsobspinto, but I think he means the ones already present, before the current integration with the upload PDF. Default ones for behavior, electrophysiology, and io_mapping. 

![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/b00e90e5-1ef1-4fae-952a-fe0e217f706d)
![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/87a46f4c-f667-4686-9c73-199b15e1fd3a)

**Comment by neuroBazzi** [comment]
@D-GopalKrishna thank you. @afonsobspinto no worries I re-uplaoded them manually. all good :-)

**Comment by afonsobspinto** [comment]
Correct @D-GopalKrishna . @neuroBazzi previously they were part of all experiments but the PDFs now are associated with a population per experiment.

In other words, this mean that if you uploaded a PDF into V3 residential population for experiment 1 it won't show in V3 residential population for experiment 2 (which makes sense as I see it).

So what you uploaded is most likely only visible for a specific experiment. We can look into ways to have 'residential' PDFs if that's something we need to provide. Let me know
